internal/graph/mintURN.go

```diff
@@ -0,0 +1,42 @@
+package graph
```

Comment:
should this just be URN, or gleanerioURN?

Reply:
do you mean the file name or the function name?

Reply:
filename

Reply:
changed this to the simpler urn.go